src/components/Calculator.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
- [ ] According to the requirements Calculator.js should have at least 1 child component